.travis.yml

```diff
@@ -24,6 +24,7 @@ jobs:
   include:
   - stage: test
     script:
+    - sbt +scalafmtCheckAll
```

Comment:
Do we need the `+` here?

Reply:
I'm not sure. We have it for `+test`, but the `travis` documentation was very ambiguous about it. Do you know for sure?

Reply:
The few examples here don't seem to have a `+argument` used, so I'm not sure why we have `+test` to begin with?

https://docs.travis-ci.com/user/languages/scala/

Reply:
We need it for `test` because we want to run the tests across all cross-compiled scala versions. It's probably not necessary here, but it doesn't hurt either.